includes/admin/home/notices/class-sensei-home-notices.php

```diff
@@ -0,0 +1,317 @@
+<?php
+/**
+ * File containing Sensei_Home_Notices class.
+ *
+ * @package sensei-lms
+ * @since   $$next-version$$
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Class that gathers and produces the local notices for Sensei Home.
```

Comment:
This and `class-sensei-home-notices-provider.php` have the same class comment, not sure if you want to specify any of them a little bit more.

Reply:
Updated `Sensei_Home_Notices_Provider`'s in 5bc61228f6370ea58128354a00c0e3c4919896e1